src/NServiceBus.AcceptanceTests/Sagas/When_a_finder_exists_and_found_saga.cs

```diff
@@ -40,7 +41,7 @@ public class CustomFinder : IFindSagas<TestSaga08.SagaData08>.Using<SomeOtherMes
                 // ReSharper disable once MemberCanBePrivate.Global
                 public Context Context { get; set; }
 
-                public Task<TestSaga08.SagaData08> FindBy(SomeOtherMessage message, ReadOnlyContextBag context)
+                public Task<TestSaga08.SagaData08> FindBy(SomeOtherMessage message, SynchronizedStorageSession storageSession, ReadOnlyContextBag context)
```

Comment:
:+1: for keeping them separate for now to make it explicit, we can collapse later if needed